DatadogInternal/Sources/NetworkInstrumentation/OpenTracing/TracingHTTPHeaders.swift

```diff
@@ -8,21 +8,21 @@ import Foundation
 
 /// Trace propagation headers as explained in
 /// https://docs.datadoghq.com/real_user_monitoring/connect_rum_and_traces/?tab=browserrum#how-are-rum-resources-linked-to-traces
-internal enum TracingHTTPHeaders {
+public enum TracingHTTPHeaders {
```

Comment:
I feel like I commented this before, but I probably missed the answer.

Does it need to be public? It seems like a Tracings internal detail 🤔

Reply:
Indeed, It's public but as part of the `DatadogInternal` module. It's necessary to make this available for the up-coming `DatadogTrace` and `DatadogRUM` modules 👍